diff --git a/node_modules/react-alert-template-basic/dist/cjs/react-alert-template-basic.js b/node_modules/react-alert-template-basic/dist/cjs/react-alert-template-basic.js
index 7d01c3d..e9f4410 100644
--- a/node_modules/react-alert-template-basic/dist/cjs/react-alert-template-basic.js
+++ b/node_modules/react-alert-template-basic/dist/cjs/react-alert-template-basic.js
@@ -80,7 +80,7 @@ var _extends = Object.assign || function (target) {
 };
 
 var alertStyle = {
-  backgroundColor: '#151515',
+  backgroundColor: '#FFFFFF',
   color: 'white',
   padding: '10px',
   textTransform: 'uppercase',
@@ -97,7 +97,7 @@ var alertStyle = {
 var buttonStyle = {
   marginLeft: '20px',
   border: 'none',
-  backgroundColor: 'transparent',
+  backgroundColor: '#FFFFFF',
   cursor: 'pointer',
   color: '#FFFFFF'
 };
diff --git a/node_modules/react-alert-template-basic/dist/esm/react-alert-template-basic.js b/node_modules/react-alert-template-basic/dist/esm/react-alert-template-basic.js
index 350f6b1..eaa09d8 100644
--- a/node_modules/react-alert-template-basic/dist/esm/react-alert-template-basic.js
+++ b/node_modules/react-alert-template-basic/dist/esm/react-alert-template-basic.js
@@ -76,7 +76,7 @@ var _extends = Object.assign || function (target) {
 };
 
 var alertStyle = {
-  backgroundColor: '#151515',
+  backgroundColor: '#FFFFFF',
   color: 'white',
   padding: '10px',
   textTransform: 'uppercase',
@@ -93,7 +93,7 @@ var alertStyle = {
 var buttonStyle = {
   marginLeft: '20px',
   border: 'none',
-  backgroundColor: 'transparent',
+  backgroundColor: '#FFFFFF',
   cursor: 'pointer',
   color: '#FFFFFF'
 };
diff --git a/node_modules/react-alert-template-basic/dist/umd/react-alert-template-basic.js b/node_modules/react-alert-template-basic/dist/umd/react-alert-template-basic.js
index 93142b9..ba0a701 100644
--- a/node_modules/react-alert-template-basic/dist/umd/react-alert-template-basic.js
+++ b/node_modules/react-alert-template-basic/dist/umd/react-alert-template-basic.js
@@ -82,7 +82,7 @@ var _extends = Object.assign || function (target) {
 };
 
 var alertStyle = {
-  backgroundColor: '#151515',
+  backgroundColor: '#FFFFFF',
   color: 'white',
   padding: '10px',
   textTransform: 'uppercase',
@@ -99,7 +99,7 @@ var alertStyle = {
 var buttonStyle = {
   marginLeft: '20px',
   border: 'none',
-  backgroundColor: 'transparent',
+  backgroundColor: '#FFFFFF',
   cursor: 'pointer',
   color: '#FFFFFF'
 };
diff --git a/node_modules/react-alert-template-basic/src/index.js b/node_modules/react-alert-template-basic/src/index.js
index bf5066d..f33c5d9 100644
--- a/node_modules/react-alert-template-basic/src/index.js
+++ b/node_modules/react-alert-template-basic/src/index.js
@@ -5,7 +5,7 @@ import ErrorIcon from './icons/ErrorIcon'
 import CloseIcon from './icons/CloseIcon'
 
 const alertStyle = {
-  backgroundColor: '#151515',
+  backgroundColor: '#FFFFFF',
   color: 'white',
   padding: '10px',
   textTransform: 'uppercase',
@@ -22,7 +22,7 @@ const alertStyle = {
 const buttonStyle = {
   marginLeft: '20px',
   border: 'none',
-  backgroundColor: 'transparent',
+  backgroundColor: '#FFFFFF',
   cursor: 'pointer',
   color: '#FFFFFF'
 }
